src/components/portfolio/DetailsMainDividends.tsx

```diff
@@ -61,43 +62,40 @@ const useStyles = makeStyles(({ palette }: Theme) =>
 
 // type declarations
 type DetailsMainDividendsProps = {
-  nextDividend: number;
-  dividendPayoutRatio: number;
+  portfolio: NonEmptyPortfolioDetails;
 };
 
 // type declarations
 type InfoBlockProps = {
   title: string;
-  body: ReactElement;
 };
 
 // returns the details page header
-const InfoBlock: React.FC<InfoBlockProps> = ({ title, body }) => {
+const InfoBlock: React.FC<InfoBlockProps> = ({ title, children }) => {
   const classes = useStyles();
 
   return (
     <div className={classes.infoWrapper}>
       <div className={classes.infoTitle}>
         <p className={classes.infoTitleP}>{title}</p>
       </div>
-      <div className={classes.infoBody}>{body}</div>
+      <div className={classes.infoBody}>{children}</div>
     </div>
   );
 };
 
 // returns the details page header
 const DetailsMainDividends: React.FC<DetailsMainDividendsProps> = ({
-  nextDividend,
-  dividendPayoutRatio,
+  portfolio,
 }) => {
   const classes = useStyles();
   const theme = useTheme();
   const { t } = useTranslation();
 
-  const mockSeries = [
+  const series = [
     {
       name: 'Dividend Yield',
```

Comment:
Translation required.

Reply:
Good point. I also noticed that the years in the line charts are hardcoded. I haven't changed it yet because it also affects the analyser team.